nav2_system_tests/src/planning/planner_tester.cpp

```diff
@@ -35,11 +35,23 @@ namespace nav2_system_tests
 {
 
 PlannerTester::PlannerTester()
-: Node("PlannerTester"), map_publish_rate_(100s), map_set_(false), costmap_set_(false),
+: Node("PlannerTester"), is_active_(false), map_set_(false), costmap_set_(false),
   using_fake_costmap_(true), costmap_server_running_(false), trinary_costmap_(true),
   track_unknown_space_(false), lethal_threshold_(100), unknown_cost_value_(-1),
-  testCostmapType_(TestCostmap::open_space), spin_thread_(nullptr)
+  testCostmapType_(TestCostmap::open_space), map_publish_rate_(100s)
 {
+}
+
+void PlannerTester::activate()
+{
+  if (is_active_) {
+    RCLCPP_WARN(this->get_logger(), "Trying to active while already active");
```

Comment:
```suggestion
    RCLCPP_WARN(this->get_logger(), "Trying to activate while already active");
```

Reply:
We both suggested the exact same thing. :)